v4/source/system.yaml

```diff
@@ -810,6 +810,28 @@
 
             // RequestTrialLicense
             resp := Client.RequestTrialLicense()
+  /trial-license/prev:
+    get:
+      tags:
+        - system
+      summary: Get last trial license used
+      description: >
+        Get the last trial license used on the sevrer
+
+        __Minimum server version__: 5.35
```

Comment:
This will probably not go in until 5.36.

Reply:
Updated the version.